pkg/helpers/azureconst.go

```diff
@@ -266,233 +266,8 @@ func GetDCOSMasterAllowedSizes() string {
 `
 }
 
-// GetMasterAgentAllowedSizes returns the agent allowed sizes
```

Comment:
Were these changes a cleanup after removing old orchestrator support, or a separate refactoring? I can't tell if this will break acs-engine or not.